monitoring/patches.yml

```diff
@@ -0,0 +1,50 @@
+# Add the following patches block to your base kustomization.yml in your fork
+# of spinnaker-config:
+
+patches:
+- target:
+    # Match all Deployments, excluding deck
+    kind: Deployment
+    name: (clouddriver|echo|fiat|front50|gate|igor|kayenta|orca|rosco)
+  path: monitoring/patches/shared/sidecar.yml
+- target:
+    # Match all Deployments, excluding deck
+    kind: Deployment
+    name: (clouddriver|echo|fiat|front50|gate|igor|kayenta|orca|rosco)
```

Comment:
There's no way to use a label selector here, right?  (From my quick search it seems like there isn't...)

Reply:
So there is a `labelSelector` field ([example](https://github.com/kubernetes-sigs/kustomize/blob/master/examples/patchMultipleObjects.md)) which I considered using, but then we'd have to add some monitoring-specific label to every Deployment except Deck upstream, which kind of seemed worse? I don't know. We also can't use any of the common Spinnaker labels we're adding in the `kustomization.yml` since the patching apparently happens before those are inserted.

Also, so glad you asked about this, because I completely forgot to tell you about the weird error I got when I tried to make a regular expression that matched every service except Deck using `?!` which said something like ["unsupported Perl syntax"](https://stackoverflow.com/questions/38933898/error-parsing-regexp-invalid-or-unsupported-perl-syntax) 😂 😂 😂

Reply:
Oh, I see there is a label selector!  But yeah I think that until we re-think this it's probably better to have this list here rather than add a monitoring-specific label upstream.

Reply:
I also think I got that error once when I forgot `kleat` was in `go` and wrote a package in `perl`... 😂